tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -617,6 +634,45 @@ EOT
 	info "Create /etc/resolv.conf file in rootfs if not exist"
 	touch "$dns_file"
 
+	if [ "${SKOPEO_UMOCI}" = "yes" ]; then
+		# TODO use versions.yaml -- see https://github.com/kata-containers/kata-containers/issues/2909 for tracking
+		skopeo_url=https://github.com/containers/skopeo
+		skopeo_branch=release-1.4
+		umoci_url=https://github.com/opencontainers/umoci
+		umoci_tag=v0.4.7
```

Comment:
Nit: Ideally these would be in the `versions.yaml` DB.

Reply:
yes, we discussed that a week ago and I opened an issue for it, see the comment :) but since it's not a few days to demo any more, I can just do it now.